### PR TITLE
Some tweaks to the documentation layout

### PR DESCRIPTION
**Pull request by davidiw**
* Removed a duplicate CONTRIBUTING.md
* Moved the contributing docs into documentation
* Moved tech-papers into documentation (we should discuss if
specifications should go in there too)

### CODE REVIEW

File: developers.libra.org/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,52 +0,0 @@
-# Contribution Guide
```
Comment (bmwill) [inline developers.libra.org/CONTRIBUTING.md:1]: I'm assuming that all of these points are already covered in the top-level one.

**Review by bmwill — APPROVED** [review]

**Comment by JoelMarcey** [comment]
Can you give me a bit to take a look at this before landing, particularly the `CONTRIBUTING.md` changes. I can after my meeting today.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ detailed guidance about how to contribute to the project.
 #### Documentation
 
 Libra's developer website is also open source (the code can be found in this
-[repository](https://github.com/libra/developers.libra.org/website/)).  It is built using
+[repository](https://github.com/libra/libra/developers.libra.org/website/)).  It is built using
 [Docusaurus](https://docusaurus.io/):
 
 If you know Markdown, you can already contribute! How to contribute can be found in the [website
```
Comment (bmwill) [inline CONTRIBUTING.md:44]: This link needs to be updated too

**Comment by JoelMarcey** [comment]
One thing I will say is that the `CONTRIBUTING.md` in `developers.libra.org` was totally different than the `CONTRIBUTING.md` in the main root. The former was specific to contributing to the website only. And was linked to, I believe, from the `CONTRIBUTING.md` in the root.

File: developers.libra.org/CONTRIBUTING.md (hunk 1 above)
Comment (davidiw) [reply developers.libra.org/CONTRIBUTING.md:1]: Yes, this is actually a really weird doc that it looks like it was intended for libra code and not website

**Comment by davidiw** [comment]
> Can you give me a bit to take a look at this before landing, particularly the `CONTRIBUTING.md` changes. I can after my meeting today.

I'm in no rush, I certainly don't want to break anything :)

My reading of the contributing doc for the website was that it was actually a smaller version of the code base.

File: developers.libra.org/CONTRIBUTING.md (hunk 1 above)
Comment (JoelMarcey) [reply developers.libra.org/CONTRIBUTING.md:1]: Whoa. I wonder if I messed something up and didn't get the right doc in this directory. Let me check.

Comment (JoelMarcey) [reply developers.libra.org/CONTRIBUTING.md:1]: No. This is the way it was on the `website` repo. So, yeah, we should remove this and maybe put a new `CONTRIBUTING.md` in `developers.libra.org` that talks about how to specifically contribute and test the docs. But we can do that later.

File: CONTRIBUTING.md (hunk 2 above)
Comment (JoelMarcey) [reply CONTRIBUTING.md:44]: I would just remove the second sentence here starting with "How to contribute...."

**Review by JoelMarcey — APPROVED** [review]
Thanks for doing this. Good catch on the duplicate `CONTRIBUTING.md`. Just a few suggestions I had, but shipit.

**Comment by bmwill** [comment]
/land

Lets get this in the queue

**Comment by JoelMarcey** [comment]
> One thing I will say is that the CONTRIBUTING.md in developers.libra.org was, I thought, totally different than the CONTRIBUTING.md in the main root. The former was specific to contributing to the website only. And was linked to, I believe, from the CONTRIBUTING.md in the root.

This was not true at all. 😬

**Comment by davidiw** [comment]
shoot pushed to the wrong commit... will fix this :D

**Pull request closed without merging**